src/test/java/com/android/volley/toolbox/AdaptedHttpStackTest.java

```diff
@@ -1,11 +1,20 @@
 package com.android.volley.toolbox;
 
-import android.util.Pair;
+import static org.junit.Assert.assertEquals;
```

Comment:
There's some inconsistency (not from this change) with whether to not to use wildcards. I vote for removing the wildcards in favor of explicit imports.

Reply:
agreed